src/metabase/util/files.clj

```diff
@@ -78,6 +78,13 @@
   [^Path path]
   (iterator-seq (.iterator (Files/list path))))
 
+(defn make-writable-parents!
```

Comment:
The implementation of this function has a strange edge case. `(make-writable-parents! ".") returns `nil`. And even though this can never happen in the serialization code, it's very strange undocumented behaviour that could cause bugs in the future.